api/src/main/java/org/openmrs/api/context/ServiceContext.java

```diff
@@ -179,9 +179,7 @@ public static void destroyInstance() {
 			}
 		}
 		
-		if (log.isDebugEnabled()) {
-			log.debug("Destroying ServiceContext instance: " + ServiceContextHolder.instance);
-		}
+			log.debug("Destroying ServiceContext instance {}", ServiceContextHolder.instance);
```

Comment:
please make it `instance:` instead of `instance`.  remember that the message that will be logged should stay the same as before your change.